pandas/core/arrays/categorical.py

```diff
@@ -1780,85 +1781,17 @@ def fillna(self, value=None, method=None, limit=None):
 
         return self._constructor(codes, dtype=self.dtype, fastpath=True)
 
-    def take(self, indexer, allow_fill: bool = False, fill_value=None):
-        """
```

Comment:
Probably want to restore this docstring.

Reply:
restored with TypeError->ValueError and whatsnew